[2023-04-13T22:39:31.535+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data_Odesa scheduled__2023-04-13T21:00:00+00:00 [queued]>
[2023-04-13T22:39:31.578+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data_Odesa scheduled__2023-04-13T21:00:00+00:00 [queued]>
[2023-04-13T22:39:31.584+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T22:39:31.589+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T22:39:31.600+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T22:39:31.717+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data_Odesa> on 2023-04-13 21:00:00+00:00
[2023-04-13T22:39:31.758+0000] {standard_task_runner.py:55} INFO - Started process 7375 to run task
[2023-04-13T22:39:31.782+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data_Odesa', 'scheduled__2023-04-13T21:00:00+00:00', '--job-id', '388', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp68wpomkg']
[2023-04-13T22:39:31.789+0000] {standard_task_runner.py:83} INFO - Job 388: Subtask inject_data_Odesa
[2023-04-13T22:39:32.063+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data_Odesa scheduled__2023-04-13T21:00:00+00:00 [running]> on host ffea11fb791c
[2023-04-13T22:39:32.503+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data_Odesa
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T21:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-13T21:00:00+00:00
[2023-04-13T22:39:32.509+0000] {sql.py:254} INFO - Executing: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Odesa",
                    1681426467,
                    282.33,
                    81,
                    "scattered clouds",
                    3.68);
            
[2023-04-13T22:39:32.545+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T22:39:32.857+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T22:39:32.862+0000] {sql.py:375} INFO - Running statement: 
                INSERT INTO measures_ext (city, timestamp, temp, humidity, cloudiness, wind_speed) VALUES
                    ("Odesa",
                    1681426467,
                    282.33,
                    81,
                    "scattered clouds",
                    3.68);
            , parameters: None
[2023-04-13T22:39:32.887+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T22:39:32.922+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data_Odesa, execution_date=20230413T210000, start_date=20230413T223931, end_date=20230413T223932
[2023-04-13T22:39:33.000+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T22:39:33.025+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
